app/javascript/react/store/mobileStreamSlice.ts

```diff
@@ -10,9 +10,7 @@ export interface MobileStreamState {
   data: MobileStream;
   status: StatusEnum;
   error?: Error;
```

Comment:
with the changes from my PR, you should use ApiError that does not allow for undefined, and nullize the error in cases other than rejected. You'll probably get the merge conflict here, let me know if you need help, (the logError will also be neeeded to cast the error to Sentry, and you'll need to update the rejectWithValue to follow ApiError